packages/x-components/tsconfig.json

```diff
@@ -12,6 +12,7 @@
     "declaration": true,
     "declarationMap": true,
     "sourceMap": true,
+    "rootDir": "src",
```

Comment:
I don't know how the build was working without the rootDir property, without it, dist types should be generated inside a `src` folder